www/include/eventLogs/xml/data.php

```diff
@@ -787,27 +797,46 @@
     foreach ($logs as $log) {
         $buffer->startElement("line");
         $buffer->writeElement("msg_type", $log["msg_type"]);
-        $displayType = $log['type'];
-        if (isset($tab_type[$log['type']])) {
-            $displayType = $tab_type[$log['type']];
+
+        /**
+         * For an ACK there is no point to display RETRY and TYPE columns
+         */
+        $displayType = '';
+        if (
+            $log['msg_type'] != HOST_ACKNOWLEDGEMENT_MSG_TYPE
+            && $log['msg_type'] != SERVICE_ACKNOWLEDGEMENT_MSG_TYPE
+        ) {
+            $displayType = $log['type'];
+
+            if (isset($tab_type[$log['type']])) {
+                $displayType = $tab_type[$log['type']];
+            }
+            $log["msg_type"] > 1 ? $buffer->writeElement("retry", "") : $buffer->writeElement("retry", $log["retry"]);
+            $log["msg_type"] == 2 || $log["msg_type"] == 3
+                ? $buffer->writeElement("type", "NOTIF")
+                : $buffer->writeElement("type", $displayType);
         }
-        $log["msg_type"] > 1 ? $buffer->writeElement("retry", "") : $buffer->writeElement("retry", $log["retry"]);
-        $log["msg_type"] == 2 || $log["msg_type"] == 3
-            ? $buffer->writeElement("type", "NOTIF")
-            : $buffer->writeElement("type", $displayType);
 
         /*
          * Color initialisation for services and hosts status
+         * For ACK message types, display a badge 'ACK' in Yellow
          */
         $color = '';
-        if (isset($log["status"])) {
-            if (
-                isset($tab_color_service[$log["status"]])
-                && !empty($log["service_description"])
-            ) {
-                $color = $tab_color_service[$log["status"]];
-            } elseif (isset($tab_color_host[$log["status"]])) {
-                $color = $tab_color_host[$log["status"]];
+        if (
+            $log['msg_type'] == HOST_ACKNOWLEDGEMENT_MSG_TYPE
+            || $log['msg_type'] == SERVICE_ACKNOWLEDGEMENT_MSG_TYPE
+        ) {
+            $color = $acknowlegementMessageType['badgeColor'];
+        } else {
+            if (isset($log["status"])) {
+                if (
+                    isset($tab_color_service[$log["status"]])
+                    && !empty($log["service_description"])
+                ) {
+                    $color = $tab_color_service[$log["status"]];
+                } elseif (isset($tab_color_host[$log["status"]])) {
+                    $color = $tab_color_host[$log["status"]];
+                }
```

Comment:
avoid deep nested conditions when you can 
```suggestion
        } elseif (isset($log["status"])) {
            if (
                isset($tab_color_service[$log["status"]])
                && !empty($log["service_description"])
            ) {
                $color = $tab_color_service[$log["status"]];
            } elseif (isset($tab_color_host[$log["status"]])) {
                $color = $tab_color_host[$log["status"]];
```